guides/migrations/0.21.0_to_0.XX.0.md

```diff
@@ -0,0 +1,69 @@
+# Migrate from version 0.21.0 to 0.XX.0
+
+## Update in `DefaultPlugin`
+
+- Merge `folder` and `fileName` params in `addComponent` method:
```

Comment:
add concrete example to let people see the difference.

Reply:
done